### PR TITLE
Update Travel Policy.md

### PR DESCRIPTION
**Pull request by saulmarit**
Added a section about how to use the company card during team trips.

### CODE REVIEW

**Comment by KataPoszler** [comment]
🗺

File: Travel Policy.md [hunk 1]
```diff
@@ -62,10 +62,14 @@ For accommodation we have previously used Airbnb-s but this might not be the cas
...
 All whole team meals during Dropsites are covered and paid by the company (Marit). This usually covers lunches and at least 3 dinners but also breakfast in some cases, depending on the Dropsite location and program. 
 
-## Other expenses during Dropsite 💳
+## Other expenses during Dropsite 💶
 
 Other expenses during Dropsite, which may include taxi rides, optional dinners, free day programs etc. are covered by the company when prepayment is possible or when either Mark, Dani or Marit are present.
 
+## Paying with company card 💳
+
+It might happen that You get the company card and are in charge of paying for the team. In case this happens please take good care of it and get a receipt or invoice for the payment, whichever is available. Usually you are given a receipt and a bank confirmation. For the accounting it is important to have the receipt with the paid sum and info (name, address) of the restaurant etc on it. You can scan the receipt and send it to invoices@languagedrops.com email with info about the cost (e.g team lunch) in a PDF format or just find Marit and give it to her.
```
Comment (fadani) [inline Travel Policy.md]: If i take a photo of the receipt and send it to you, should I still keep the original one and give you the next time we are at the same place?

Comment (saulmarit) [reply Travel Policy.md]: Thank you for the question! No need to keep the original in that case. 😃

Comment (CHR15-) [reply Travel Policy.md]: Are we using receipt bank @saulmarit ?

Comment (saulmarit) [reply Travel Policy.md]: No, at the moment we do not use any software for that, good old manual work 😅 But we might in the future and then I can initiate my second ever pull request! 🎉

**Comment by tincuyugan** [comment]
can't approve as i was requested to review but just leaving this comment to acknowledge i've read & understood it! :)

File: Travel Policy.md (hunk 1 above)
Comment (CHR15-) [reply Travel Policy.md]: sure, it makes expenses ultra easy for all :D

**Pull request merged**